packages/camera/android/src/main/java/io/flutter/plugins/camera/media/MediaRecorderBuilder.java

```diff
@@ -49,18 +49,18 @@ public MediaRecorderBuilder setMediaOrientation(int orientation) {
   public MediaRecorder build() throws IOException {
     MediaRecorder mediaRecorder = recorderFactory.makeMediaRecorder();
 
-    // There's a specific order that mediaRecorder expects. Do not change the order
-    // of these function calls.
+    // There's a fixed order that mediaRecorder expects. Only change these functions accordingly.
+    // You can find the specifics here: https://developer.android.com/reference/android/media/MediaRecorder.
```

Comment:
Thanks for the link! it makes more sense now.

nit: it doesn't seem like the requirement is for a specific fixed ordering but that in each media recorder state a different set of calls are allowed (e.g we can replace the ordering for setAudioSource and setVideoSource).

Maybe the comment should say something like:

```
The setters below drive the internal state machine of the media recorder, these setters cannot be called in any ordering but must satisfy the state constraints detailed in: https://developer.android.com/reference/android/media/MediaRecorder.
```